src/App/App.xaml.cs

```diff
@@ -31,6 +31,7 @@ public partial class App : Application, IAccountsManagerHost
         private readonly IAccountsManager _accountsManager;
         private readonly IPushNotificationService _pushNotificationService;
         private static bool _isResumed;
+        private static bool _pendingCheckPasswordlessLoginRequests;
```

Comment:
why is this `static`?

Reply:
otherwise the value gets lost on sleep on android.